tools/downloader/pytorch_to_onnx.py

```diff
@@ -65,6 +81,8 @@ def parse_args():
                         help='Comma-separated names of the output layers')
     parser.add_argument('--model-param', type=model_parameter, default=[], action='append',
                         help='Pair "name"="value" of model constructor parameter')
+    parser.add_argument('--inputs-dtype', type=str, required=False, choices=INPUT_DTYPE_TO_TORCH,
+                        help='Data type for inputs')
```

Comment:
I would suggest to add `default='float'`, which would simplify the usage quite a bit (I don't think we need to support the `dtype=None` behavior).

Reply:
agree, added default